server/review.go

```diff
@@ -20,6 +20,12 @@ import (
 const mlogReviewCommentBody = "Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change."
 
 func (s *Server) reviewMlog(ctx context.Context, pr *model.PullRequest, nodeID, diffURL string) error {
+	for _, m := range s.OrgMembers {
```

Comment:
i think you can use `s.IsOrgMember(pr.Username)`

Reply:
Thanks for pointing that one, I also added a comment.